client/branded/src/search-ui/components/ResultContainer.tsx

```diff
@@ -35,6 +36,25 @@ const accessibleResultType: Record<SearchMatch['type'], string> = {
     team: 'team',
 }
 
+const RepoKVPs: React.FunctionComponent<{ keyValuePairs?: Record<string, string>; className?: string }> = ({
+    keyValuePairs,
+    className,
+}) => {
+    if (!keyValuePairs) {
+        return null
+    }
+    return (
+        <div className={classNames(styles.repoKvps, className, 'd-flex align-items-center flex-wrap')}>
```

Comment:
same as above, I think it might be better to name this `styles.repoMeta` or similar:
```suggestion
        <div className={classNames(styles.repoMeta, className, 'd-flex align-items-center flex-wrap')}>
```